### ISSUE TITLE
`toolbox: missing command` is confusing

### ISSUE THREAD

**Issue by allanday** [debarshiray/toolbox#124]
If I run `toolbox`, the response is `toolbox: missing command`. This is confusing: it sounds like bash is telling me that toolbox isn't installed.

A better response would be to list the available commands or, for a cheaper approach, just say:

```
To see available toolbox commands, run:

    toolbox --help
```

**Comment by zerotri** [comment]
Which version of toolbox are you running? As of now the latest version in git spits out:
```
toolbox: missing command

These are some common commands:
create    Create a new toolbox container
enter     Enter an existing toolbox container
list      List all existing toolbox containers and images

Try 'toolbox --help' for more information.
```
Which is a little more helpful than just `missing command`. I do agree that the phrase `missing command` is a little confusing here as if I misspell toolbox for some reason, the output from bash would be: `bash: tolobox: command not found`.

**Comment by debarshiray** [comment]
Thanks for triaging this, @zerotri 

Yes, this was covered in PR #112 in response to issue #59  The changes are included in release 0.0.8 but it's still stuck in the respective `updates-testing` repositories. In the meantime, you can use the script directly from Git (eg., `$ ./toolbox --help`) or try the following steps to grab the RPM from `updates-testing`.

On a Fedora 29 host:
```
$ wget https://kojipkgs.fedoraproject.org//packages/toolbox/0.0.8/2.fc29/noarch/toolbox-0.0.8-2.fc29.noarch.rpm
```

Then, on a legacy Workstation:
```
$ rpm -Uvh toolbox-0.0.8-2.fc29.noarch.rpm
```
On Silverblue:
```
$ rpm-ostree install toolbox-0.0.8-2.fc29.noarch.rpm
$ reboot
```

On a Fedora 30 host, replace the first command with the following and adjust the subsequent commands accordingly:
```
$ wget https://kojipkgs.fedoraproject.org//packages/toolbox/0.0.8/2.fc30/noarch/toolbox-0.0.8-2.fc30.noarch.rpm
```

**Comment by debarshiray** [comment]
Duplicate of #59
